lib/galaxy/datatypes/converters/interval_to_bgzip_converter.xml

```diff
@@ -29,27 +40,54 @@
     <tests>
         <test>
             <param name="input1" ftype="bed" value="droPer1.bed"/>
-            <output name="output1" ftype="bgzip" value="droPer1.bgzip"/>
+            <output name="output1" ftype="bgzip" value="droPer1.bgzip" decompress="true">
+                <expand macro="assert_size" value="131"/>
+            </output>
+            <assert_command>
+                <has_text text="-k1,1 -k2,2n -k3,3n"/>
+            </assert_command>
         </test>
         <test>
             <param name="input1" ftype="encodepeak" value="encode.broad.peak"/>
             <output name="output1" ftype="bgzip">
-                <assert_contents>
-                    <has_size value="299" delta="10"/>
-                </assert_contents>
+                <expand macro="assert_size" value="143">
+                    <has_text_matching expression="^#" negate="true"/>
+                </expand>
             </output>
+            <assert_command>
+                <has_text text="-k1,1 -k2,2n -k3,3n"/>
+            </assert_command>
         </test>
         <test>
             <param name="input1" ftype="gff" value="gff_filter_by_feature_count_out2.gff"/>
-            <output name="output1" ftype="bgzip" value="bgzip_filter_by_feature_count_out2.bgzip"/>
+            <output name="output1" ftype="bgzip" value="bgzip_filter_by_feature_count_out2.bgzip" decompress="true">
+                <expand macro="assert_size" value="638"/>
+            </output>
+            <assert_command>
+                <has_text text="-k1,1 -k4,4n"/>
+                <has_text text="-k5,5" negate="true"/>
+            </assert_command>
         </test>
         <test>
             <param name="input1" ftype="interval" value="2.interval"/>
-            <output name="output1" ftype="bgzip" value="2.bgzip"/>
+            <output name="output1" ftype="bgzip" value="2.bgzip" decompress="true">
+                <expand macro="assert_size" value="208"/>
+            </output>
+            <assert_command>
+                <has_text text="-k1,1 -k2,2n -k3,3n"/>
+            </assert_command>
         </test>
         <test>
             <param name="input1" ftype="vcf" value="vcf_to_maf_in.vcf"/>
-            <output name="output1" ftype="bgzip" value="bgzip_to_maf_in.bgzip"/>
+            <output name="output1" ftype="bgzip" decompress="true">
+                <expand macro="assert_size" value="322">
+                    <has_n_lines n="3"/>
```

Comment:
This is failing with:
```
Output output1:  different than expected
Expected 3+-0 lines in the output found 7
```

Although this should be 5?

```shell
$ grep -e '^#' -e '^track' -v vcf_to_maf_in.vcf | sort -k1,1 -k2,2n | bgzip -c | gunzip | wc -l
5
```

Reply:
Forgot that https://github.com/galaxyproject/galaxy/pull/15085 has not been accepted yet. So the assertions are still running on the compressed content. Guess that is the reason. 

I could restore `value="bgzip_to_maf_in.bgzip" decompress="true"`, but then we need to update the test data which is also not ideal.

Reply:
Maybe drop this line for now then?

Reply:
Maybe drop this line for now then?

```suggestion
```